src/fsharp/CompileOptions.fs

```diff
@@ -662,6 +674,13 @@ let advancedFlagsBoth tcConfigB =
         utf8OutputFlag tcConfigB;
         fullPathsFlag tcConfigB;
         libFlag tcConfigB;
+        CompilerOption("preferreduilang", tagString, OptionString (fun s -> 
```

Comment:
@enricosada Is there a reason why you written this option inline? For consistency reasons I would provide it as an own function like the other flags in this list.